framework/include/MonteCarloPlayerMT.h

```diff
@@ -0,0 +1,47 @@
+#ifndef _MONTECARLOPLAYERMT_H
+#define _MONTECARLOPLAYERMT_H
+
+#include <vector>
+#include <mutex>
+#include <atomic>
+#include <condition_variable>
+#include <thread>
+#include <functional>
+
+#include "MonteCarloPlayer.h"
+
+#define NUM_END_STATES_DESIRED 25
+#define NUM_THREADS 4
+#define EXPLORATION_PARAM 1
```

Comment:
All instances of EXPLORATION_PARAM and ITERATION_COUNT should either be in the CPP, named differently, or defined using an enum so they do not cause warnings. If an enum is used, it could be put in the MonteCarloTypes.h file.

Reply:
good catch. renamed in next commit